### PR TITLE
Docker and docker-compose setup.

### LINKED ISSUE #24: Dockerize the project for vendor agnostic distribution
Task
- [ ] Implement initial dockerfile

### PR DESCRIPTION
**Pull request by sudiptob2**
closes #24

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+
+RUN pip install pipenv
+RUN pipenv install --system --deploy --dev
+
+# Make port 5000 available to the world outside this container
+EXPOSE 5000
+
+# Run app.py when the container launches
+ENTRYPOINT ["flask", "run" , "--host=0.0.0.0"]
```
Comment (Pradyuman7) [inline Dockerfile:18]: This looks good, can you also add the instructions to run the docker container in the readme?

Comment (sudiptob2) [reply Dockerfile:18]: Sure

Comment (sudiptob2) [reply Dockerfile:18]: updated

**Review by Pradyuman7 — APPROVED** [review]
LGTM

**Pull request merged**